src/components/export-gist.jsx

```diff
@@ -0,0 +1,41 @@
+/* globals XMLHttpRequest:false window:false */
```

Comment:
The [documentation](http://eslint.org/docs/user-guide/configuring#specifying-globals) states the singular `global`; might be that the plural is an alias